configs/components/puppet.json

```diff
@@ -1 +1 @@
-{"url":"git://github.com/puppetlabs/puppet.git","ref":"5846904f9f5c9ea65c87d51d5e0dd6d087b48bff"}
+{"url":"git://github.com/puppetlabs/puppet.git","ref":"7435bf2a6504c01d0c9cf57c83bb62bf6f80b842"}
```

Comment:
don't we want to avoid merging these up?

Reply:
Yes, but 7435bf2a is the current ref in master: https://github.com/puppetlabs/puppet-agent/commit/83c757c4006e7f501da0dd61a61c6faf0dbe88c4 - I'm not exactly sure why this is showing up as a diff...

Reply:
Maybe it's that you did this merge up before `7435bf2a6504c01d0c9cf57c83bb62bf6f80b842` was promoted? that is strange...

but yes, `7435bf2a6504c01d0c9cf57c83bb62bf6f80b842` is the current ref in master and `5846904f9f5c9ea65c87d51d5e0dd6d087b48bff` was the previous ref. huh. Well, I'm fine with merging this as is